src/components/ComingSoon/ComingSoon.css

```diff
@@ -0,0 +1,90 @@
+/* 
+https://brandpalettes.com/google-colors/ I decided to use brand colours of google to keep theme consistent - A.N.
+CSS Style inspired by Bold by Rafal Tomal https://wdexplorer.com/20-examples-beautiful-css-typography-design/
+*/
+
+:root {
+  --google-blue: #4285f4;
+  --google-red: #db4437;
+  --google-yellow: #f4b400;
+  --google-green: #0f9d58;
+  --google-white: #ffff;
+}
+
+/* Brand Colours */
+.text-google-blue {
+  color: var(--google-blue);
+}
+
+.text-google-red {
+  color: var(--google-red);
+}
+
+.text-google-green {
+  color: var(--google-green);
+}
+.text-google-yellow {
+  color: var(--google-yellow);
+}
+
+.text-google-white {
+  color: var(--google-white);
+}
+
+
+
+/* Text-Align & Center */
+.center {
+  margin: auto;
+  width: 60%;
+  border: 5px solid var(--google-blue);
```

Comment:
I'm not a big fan of this border, I think it's better to just add a bit more margin around the box

Reply:
I removed the border. I wasn't able to get margin to increase without white space occurring but I added padding and set margins for left and right. 


<img width="365" alt="Screen Shot 2022-02-09 at 10 51 00 AM" src="https://user-images.githubusercontent.com/74369791/153237637-a39338b2-e502-4e18-8710-415698579e49.png">
<img width="891" alt="Screen Shot 2022-02-09 at 10 51 42 AM" src="https://user-images.githubusercontent.com/74369791/153237776-c37e0edf-d683-4f81-ba6f-5838253fbe86.png">

Reply:
Awesome!